### PR TITLE
middle difelity

### PR DESCRIPTION
**Pull request by AndreLau11**
- Add the classes method for objects on javascript
- Add the style for the middle fidelity wireframe

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,73 +1,63 @@
...
 
+- Major languages: HTML, CSS
+- Technologies used: Lighthouse, Webhint, Stylelint
 
-## Getting Started
+## Live Demo
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+[Live Demo Link](https://kaitcham.github.io/Portfolio/)
```
Comment (ShoiraTa) [inline README.md]: - [x] Please make sure to update the link, you still have Portfolio project link here 🙈

File: README.md [hunk 2]
```diff
@@ -1,73 +1,63 @@
...
+- **To clone the main project**
 
-### Usage
-
-### Run tests
-
-### Deployment
+```
+git clone https://github.com/kaitcham/Portfolio
+```
```
Comment (ShoiraTa) [inline README.md]: - [x] Please update the link here as well

File: README.md [hunk 3]
```diff
@@ -1,73 +1,63 @@
...
 
+- **To clone the mobile-skeleton branch**
 
+```
+git clone -b mobile-skeleton https://github.com/kaitcham/Portfolio/mobile-skeleton
+```
```
Comment (ShoiraTa) [inline README.md]: - [x] And here 😁

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @AndreLau11 ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

**First let me to point out:**

- Project is properly setup! ✔️ 
- Amazing UI! ✔️ 
- Using JS classes! ✔️ 

Keep on coding, good job so far! :clap:

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

- [ ] Please make sure to setup linter properly

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Deepakdanger — APPROVED** [review]
Hi TEAM, 

### Status: Approved ✔️

Great work on making the changes requested by a previous reviewer 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [x] Please resolve the linter issue by updating the linter files.Hope this will work

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**